packaging/version_info

```diff
@@ -1,2 +1,2 @@
-export CLOUDIFY_CLUSTER_MANAGER_VERSION=1.0
+export CLOUDIFY_CLUSTER_MANAGER_VERSION=1.0.1
 export CLOUDIFY_CLUSTER_MANAGER_PACKAGE_RELEASE=.dev1
```

Comment:
@YoniItzhak is this something you need to have it here `.dev1`

Reply:
That's a great question. The only place I use it is here https://github.com/cloudify-cosmo/cloudify-cluster-manager/blob/0c8784c7f8574bc2859bf2f5b5fee03b84394bc8/jenkins/Jenkinsfile#L8. So maybe I should just define the `PRERELEASE` env. variable to always be `.dev1`?

Reply:
I mean is this related to our .dev1 release or something specific to cluster script ?

Reply:
I can see that .dev1 is always be .dev1 ? maybe no need to have it and store the release directly instead of .dev1 ? because cannot see the value of it since its seperaate from our pipeline ?

Reply:
Right, I'll rewrite it. Thanks!